### PR TITLE
Update README.md and Security.md and fix vulnerabilities

### PR DESCRIPTION
**Pull request by skanda890**
## Summary
Update README.md to remove spelling mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ These GitHub repositories provide supplementary resources for Rush Stack:
 | [/build-tests/rush-lib-declaration-paths-test](./build-tests/rush-lib-declaration-paths-test/) | This project ensures all of the paths in rush-lib/lib/... have imports that resolve correctly. If this project builds, all `lib/**/*.d.ts` files in the `@microsoft/rush-lib` package are valid. |
 | [/build-tests/rush-project-change-analyzer-test](./build-tests/rush-project-change-analyzer-test/) | This is an example project that uses rush-lib's ProjectChangeAnalyzer to  |
 | [/build-tests/rush-redis-cobuild-plugin-integration-test](./build-tests/rush-redis-cobuild-plugin-integration-test/) | Tests connecting to an redis server |
-| [/build-tests/set-webpack-public-path-plugin-test](./build-tests/set-webpack-public-path-plugin-test/) | Building this project tests the set-webpack-public-path-plugin |
+| [/build-tests/set-webpack-public-path-plugin-test](./build-tests/set-web pack-public-path-plugin-test/) | Building this project tests the set-webpack-public-path-plugin |
```
Comment (MartinJohns) [inline README.md:187]: Do you even check the changes you propose? Or is this based on some automatic tool?

File: README.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ These GitHub repositories provide supplementary resources for Rush Stack:
 | [/build-tests/heft-parameter-plugin](./build-tests/heft-parameter-plugin/) | This project contains a Heft plugin that adds a custom parameter to built-in actions |
 | [/build-tests/heft-parameter-plugin-test](./build-tests/heft-parameter-plugin-test/) | This project exercises a built-in Heft action with a custom parameter |
 | [/build-tests/heft-sass-test](./build-tests/heft-sass-test/) | This project illustrates a minimal tutorial Heft project targeting the web browser runtime |
-| [/build-tests/heft-typescript-composite-test](./build-tests/heft-typescript-composite-test/) | Building this project tests behavior of Heft when the tsconfig.json file uses project references. |
+| [/build-tests/heft-typescript-composite-test](./build-tests/heft-typescript-composite-test/) | Building this project tests the behavior of Heft when the tsconfig.json file uses project references. |
```
Comment (iclanton) [inline README.md:169]: This portion of this file is automatically generated. To update this, you'll also need to update the description field of that project's package.json.

**Review by iclanton — COMMENTED** [review]
You need to run `rush update` after changing dependency versions.

**Comment by skanda890** [comment]
> You need to run `rush update` after changing dependency versions. 

Is it for all the files?

**Review by skanda890 — COMMENTED** [review]
Is this fine?

**Pull request closed without merging**